build.sh

```diff
@@ -24,7 +24,7 @@ fi
 # Note that lsof returns a non-zero exit code if there is no match.
 SANDBOX_PID="$(lsof -ti tcp:6865 || true)"
 if [ -n "$SANDBOX_PID" ]; then
-    kill "$SANDBOX_PID"
+    echo "$SANDBOX_PID" | xargs kill
```

Comment:
I think we can also just remove the quotes? If we get so many pids that we need xargs other functionalites we have a problem. That said, I guess there is no harm in doing that even if we never need it.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/9293/commits/fe45bb8e0542dcb45881d642a7c3e6273d9b6e07